README.md

```diff
@@ -154,7 +166,7 @@ Create task manifest for writing to a file. You can also use an asterisk - see [
 ```
 Create a task by the following command:
 ```
-$ $SNAP_PATH/bin/snapctl task create -t ../snap-plugin-collector-docker/examples/tasks/docker-file.json
+$ snapctl task create -t examples/tasks/docker-file.json
 
 Using task manifest to create task
```

Comment:
"Task Manifest" is a term in the Snap glossary. Please capitalize wherever you use it (task can remain lowercase).